build/build-extensions.ps1

```diff
@@ -241,18 +238,25 @@ function CreateSiteExtensions() {
     Write-Host
     
     $zipOutput = "$buildOutput\SiteExtension"
+    $hashesForHardLinksPath = "$siteExtensionPath\$extensionVersion\$hashesForHardlinksFile"
     New-Item -Itemtype directory -path $zipOutput -Force > $null
     if ($minorVersionPrefix -eq "10") {
         ZipContent $siteExtensionPath "$zipOutput\Functions.$extensionVersion$runtimeSuffix.zip"
     } elseif ($minorVersionPrefix -eq "8") {
+        # Only the "Functions" site extension supports hard links
+        Write-Host "Removing $hashesForHardLinksPath before zipping."
+        Remove-Item -Force "$hashesForHardLinksPath" -ErrorAction Stop
         # The .NET 8 host doesn't require any workers. Doing this to save space.
-        Write-Host "Removing workers."
+        Write-Host "Removing workers before zipping."
         Remove-Item -Recurse -Force "$siteExtensionPath\$extensionVersion$runtimeSuffix\workers" -ErrorAction Stop
         # The host requires that this folder exists, even if it's empty
-        New-Item -Itemtype directory -path $siteExtensionPath\$extensionVersion$runtimeSuffix\workers
+        New-Item -Itemtype directory -path $siteExtensionPath\$extensionVersion$runtimeSuffix\workers > $null 
         Write-Host
         ZipContent $siteExtensionPath "$zipOutput\FunctionsInProc8.$extensionVersion$runtimeSuffix.zip"
     } elseif ($minorVersionPrefix -eq "6") {
+        # Only the "Functions" site extension supports hard links
```

Comment:
Do we not need `hashesForHardLinks` temporarily still? "6" remains in Functions at the moment. Or I guess we're saying they only become available once "10" is firmly in place? Without the `hardlinksCreated` marker. I guess we don't need the hashes to be provided?

Reply:
`hashesforhardlinks` is the signal to Antares that this extension supports hard links and it should wait for the `hardlinkscreated` file to exist.

so we could either:
1. artificially create that `hardlinkscreated` file (this is what we'd been doing in Functions)
2. not include the `hashesforhardlinks` at all

number 1 is more correct so i'm opting for deleting that file from FunctionsInProc and FunctiosnInProc8 and only leaving it for Functions (which is all that's supported)